ci/windows_ci_setup.ps1

```diff
@@ -6,7 +6,14 @@ mkdir "$env:TOOLS_BIN_DIR"
 
 $wc = New-Object System.Net.WebClient
 $wc.DownloadFile("https://github.com/bazelbuild/bazelisk/releases/download/v1.0/bazelisk-windows-amd64.exe", "$env:TOOLS_BIN_DIR\bazel.exe")
+$wc.DownloadFile("https://github.com/ninja-build/ninja/releases/download/v1.9.0/ninja-win.zip", "$env:TOOLS_BIN_DIR\ninja-win.zip")
```

Comment:
Is there any way to pin to a SHA256 here for verification?

Reply:
Added.